Xwt.Gtk/Xwt.GtkBackend/PanedBackend.cs

```diff
@@ -92,15 +88,34 @@ public void UpdatePanel (int panel, bool resize, bool shrink)
 			get { return Widget.Position; }
 			set { Widget.Position = (int) value; }
 		}
-
-		public Size GetDecorationSize ()
+		
+		public override void EnableEvent (object eventId)
 		{
-			throw new NotSupportedException ();
+			base.EnableEvent (eventId);
+			if (eventId is PanedEvent) {
+				switch ((PanedEvent)eventId) {
+					case PanedEvent.PositionChanged:
+						Widget.AddNotification ("position", HandleMove);break;
+				}
+			}
+		}
+		
+		public override void DisableEvent (object eventId)
+		{
+			base.DisableEvent (eventId);
+			if (eventId is PanedEvent) {
+				switch ((PanedEvent)eventId) {
+					case PanedEvent.PositionChanged:
+						Widget.RemoveNotification ("position", HandleMove);break;
+				}
+			}
 		}
 
-		public void GetPanelSizes (double totalSize, out double panel1Size, out double panel2Size)
+		void HandleMove (object o, GLib.NotifyArgs args)
 		{
-			throw new NotSupportedException ();
+			ApplicationContext.InvokeUserCode (delegate {
```

Comment:
The handler is connected to the "position" property change event only (notification subscription in [Line 98](https://github.com/mono/xwt/pull/472/files#diff-58e52c205c8806a597fecc8b172941e3R98)). So the check would be redundant. A property name check would make sense, if the handler would handle notifications for more then one property.